sdk/core/System.Net.ClientModel/tests/client/OpenAIClient/Choice.Serialization.cs

```diff
@@ -74,7 +79,17 @@ internal static Choice DeserializeChoice(JsonElement element)
         internal static Choice FromResponse(PipelineResponse response)
         {
             using var document = JsonDocument.Parse(response.Content);
-            return DeserializeChoice(document.RootElement);
+            return DeserializeChoice(document.RootElement, ModelReaderWriterOptions.DefaultWireOptions);
```

Comment:
nit: since we will dispose document at the end of this method, is there any way we can programmatically ensure that the `DeserializeChoice` method never assumes it has ownership of the document.RootElement JsonElement?

Reply:
not in C#